README.md

```diff
@@ -1,22 +1,126 @@
-# mat-dp-pipeline
+# Mat DP Pipeline
```

Comment:
Would you mind removing examples/notebook.ipynb if we are not going down that route? It's stale anyway..